conanfile_rtc.py

```diff
@@ -0,0 +1,25 @@
+from conans import ConanFile
+
+
+class glfwConan(ConanFile):
+    name = "glfw"
+    version = "3.3.8"
+    url = "https://github.com/Esri/glfw"
+    license = "https://github.com/Esri/glfw/LICENSE.md"
```

Comment:
The link doesn't work. Needs to be `https://github.com/Esri/glfw/blob/master/LICENSE.md`